src/targets/gpu/convolution.cpp

```diff
@@ -9,15 +9,38 @@ namespace gpu {
 shape miopen_convolution::compute_shape(const std::vector<shape>& inputs) const
 {
     check_shapes{inputs, *this}.has(4).standard();
-    return op.compute_shape({inputs.at(0), inputs.at(1)});
+    std::vector<shape> conv_inputs(inputs.begin(), inputs.begin() + 2);
+    check_shapes{conv_inputs, *this}.max_ndims(5);
+    return op.compute_shape(conv_inputs);
 }
+
+inline void recompute_shape_to_2d(shape& input)
```

Comment:
This should return the new shape instead of modifying it.

Reply:
fixed